bigquery/google/cloud/bigquery/job.py

```diff
@@ -759,6 +783,28 @@ def _scrub_local_properties(self, cleaned):
         schema = cleaned.pop('schema', {'fields': ()})
         self.schema = _parse_schema_resource(schema)
 
+    def _copy_configuration_properties(self, configuration):
+        """Helper:  assign subclass configuration properties in cleaned."""
+        self.allow_jagged_rows = _bool_or_none(
+            configuration.get('allowJaggedRows'))
+        self.allow_quoted_newlines = _bool_or_none(
+            configuration.get('allowQuotedNewlines'))
+        self.autodetect = _bool_or_none(
+            configuration.get('autodetect'))
+        self.create_disposition = configuration.get('createDisposition')
+        self.encoding = configuration.get('encoding')
+        self.field_delimiter = configuration.get('fieldDelimiter')
+        self.ignore_unknown_values = _bool_or_none(
+            configuration.get('ignoreUnknownValues'))
+        self.max_bad_records = _int_or_none(
+            configuration.get('maxBadRecords'))
+        self.null_marker = configuration.get('nullMarker')
+        self.quote_character = configuration.get('quote')
+        self.skip_leading_rows = _int_or_none(
```

Comment:
When this is sent to the API, skipLeadingRows needs to be converted back into a string.

    "skipLeadingRows": {
     "type": "string",
     "description": "[Optional] The number of rows at the top of a CSV file that BigQuery will skip when reading the data. The default value is 0. This property is useful if you have header rows in the file that should be skipped.",
     "format": "int64"
    }

from https://www.googleapis.com/discovery/v1/apis/bigquery/v2/rest

Reply:
7b2fb70fed78c7dbed283bb2f9cd3546dee6d714